contracts/permissionless/OrderbookReserve.sol

```diff
@@ -475,7 +500,7 @@ contract OrderbookReserve is OrderIdManager, Utils2, KyberReserveInterface, Orde
         view
         returns (uint32)
     {
-        require(srcAmount >= minNewOrderSizeWei);
+        require(srcAmount >= limits.minNewOrderSizeWei);
```

Comment:
this function is a bit weird, it actually validates the input orderId, not gets it.
maybe change the name? (also for the one above)

Reply:
name changed for both.